app/views/landing_page/blocks/_press_notices.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if block.items.any? %>
+  <%= render "govuk_publishing_components/components/heading", {
+    text: "Press Notices",
+    padding: true,
+    border_top: 5,
+    margin_bottom: 3
+  } %>
+
+  <%= render "govuk_publishing_components/components/document_list", {
+    margin_bottom: 4,
```

Comment:
I think you could take this margin off (or probably set it to zero) as we now have a general spacing model for blocks, which will put in the required space beneath this block.

Reply:
Margin is now set to 0 to work with the general spacing.